federation/planner.go

```diff
@@ -104,7 +104,7 @@ func printSelections(selectionSet *graphql.SelectionSet) {
 		fmt.Println(" selections")
 		for _, subSelection := range selectionSet.Selections {
 			fmt.Println(" ", subSelection.Name)
-			if (subSelection.Args != nil) {
+			if subSelection.Args != nil {
```

Comment:
Do we want to error out at line 161 if `serviceWithField` is empty?

Reply:
This is already verified when we parse the query, but we can definitely also add an error check on line 161 too